db/sqlexplain.c

```diff
@@ -189,14 +190,18 @@ static int print_cursor_description(strbuf *out, struct cursor_info *cinfo)
             strbuf_appendf(out, "index ");
         }
         strbuf_appendf(out, "temp_%d", cinfo->tbl);
+    } else if (cinfo->tbl < 0) {
+        strbuf_append(out, "table \"???\"");
     } else {
+        if (!explain)
+            rdlock_schema_lk();
```

Comment:
Currently regrabbing lock on each relevant op code so don't have schema lock for too long. Should I keep it here, or should I put it around the for loop in db_query_plan.c that iterates through all op codes?